tests/test_api_build.py

```diff
@@ -781,15 +780,6 @@ def test_noarch_python_1(testing_config):
     assert 'package_metadata_version' in extra
 
 
-@pytest.mark.sanity
-@pytest.mark.xfail(conda_47, reason="parallel verify/execute in conda 4.7 breaks legacy noarch, which depends on having the env files present before pre-link scripts are run.")
```

Comment:
Should only remove the `@pytest.mark.xfail` but not the whole test case.

Reply:
so rewrite the test to check for an expected exception?

Reply:
Nvm, I somehow read it as `not conda_47`, sorry. Okay to remove then :).